canonical-web-and-design/jaas-dashboard
# Update the UI of the Juju Web CLI

## Done

- Update the UI of the Juju Web CLI to the latest design.
- Removed the old `Terminal` component and dependencies.

## QA

- Bootstrap a new controller using the Juju 2.9 snap
  - `snap refresh --channel latest/beta`
- Build this PR and follow [this guide to connect to your controller](https://github.com/canonical-web-and-design/jaas-dashboard#developing-while-connected-to-a-juju-controller)
- View your controller model and the terminal should appear along the bottom.
- Steps:
  - - [ ] Click the help icon, did it show the help?
  - - [ ] Type `status` in the prompt and submit, did the model status appear?
  - - [ ] Type `status --color` in the prompt and submit, did the old status get cleared and a new coloured one appear?
  - - [ ] Type `help commands` in the prompt and submit, can you scroll the output
  - - [ ] Click and drag along the top of the output, does it resize as expected
  - - [ ] Drag it to the bottom so it's "closed"
  - - [ ] Type `status` in the prompt and submit, did it re-open?
  - - [ ] Click the help icon, did it resize and show the help content?
  - - [ ] Chaos monkey the interactions, can you find any bugs? Improvements?
  - - [ ] Chaos monkey the juju commands, can you find any bugs? Improvements?

[Zeplin Designs](https://app.zeplin.io/project/5f6234f4cb1e87aac4650d25/dashboard?sid=5f6235582ba7c6a777814c7f)

## Details

Fixes https://github.com/canonical-web-and-design/juju-squad/issues/1445
Fixes https://github.com/canonical-web-and-design/juju-squad/issues/1446
Fixes https://github.com/canonical-web-and-design/juju-squad/issues/1447
Fixes https://github.com/canonical-web-and-design/juju-squad/issues/1448
Fixes https://github.com/canonical-web-and-design/juju-squad/issues/1456
Fixes https://github.com/canonical-web-and-design/juju-squad/issues/1396

## Screenshots
![Screen Shot 2020-10-21 at 12 07 20 PM](https://user-images.githubusercontent.com/532033/96788504-da259500-13b0-11eb-990f-87afaa1450b1.png)
